gtk4/src/bool_filter.rs

```diff
@@ -0,0 +1,26 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{BoolFilter, Expression};
+use glib::translate::*;
+
+impl BoolFilter {
+    #[doc(alias = "gtk_bool_filter_new")]
+    pub fn new<E: AsRef<Expression>>(expression: Option<&E>) -> BoolFilter {
```

Comment:
It's going to be a bit annoying to pass in `None` here, just like with `IsA`. Maybe you want to add a `NONE_EXPRESSION` constant in `expression.rs`